### PR TITLE
Drawer refactor

### PR DESCRIPTION
**Pull request by OliWalker**
Fully refactored the drawer, 
added extra animations here and there, 
stitched up the last few http request,

### CODE REVIEW

File: versus-fe/src/App.js [hunk 1]
```diff
@@ -17,15 +17,15 @@ import { getUserInfo } from './redux/actions';
 import Drawer from './components/Drawer/Drawer';
 
 class App extends Component {
-  constructor(props) {
-    super(props);
+    componentDidMount() {
```
Comment (arol) [inline versus-fe/src/App.js]: Indentation.

File: versus-fe/src/components/Match/innerMatch.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      <div className="MatchContainer__Content__waiting">
+        <h2>
+          <i>waiting...</i>
+        </h2>
+      </div>
+    );
+  },
+
+  denied: props => {
+    console.log(props);
```
Comment (arol) [inline versus-fe/src/components/Match/innerMatch.js]: Remove console.logs, your devtools console is going to be a mess after some time,.

File: versus-fe/src/App.js (hunk 1 above)
Comment (OliWalker) [reply versus-fe/src/App.js]: fixed

File: versus-fe/src/components/Match/innerMatch.js (hunk 2 above)
Comment (OliWalker) [reply versus-fe/src/components/Match/innerMatch.js]: fixed

**Pull request merged**